pkg/controllers/reconciler.go

```diff
@@ -175,6 +176,15 @@ func UpgradeImages(image *string, initimage *string) (upgrade bool) {
 
 		if parts := strings.SplitN(*s, ":", 2); len(parts) == 2 && len(parts[0]) > 0 {
 			name, version := parts[0], parts[1]
+
+			envVarValue := os.Getenv(strings.ToUpper(name) + "_SHA")
```

Comment:
this means the value becomes, e.g, `INTEL-SGX-PLUGIN_SHA`. better make all underscores?

Reply:
yeah, I have thought about this matter, but did not have any idea what is better.
If you think so, let me make all hyphens become underscores.

Reply:
Hyphens are not supported characters for environment variable names.  They might work in some shell versions, in some situations (e.g. if shell variable name is given in another shell variable), but not generally.